docs/components/content/examples/SlideoverExampleComponent.vue

```diff
@@ -1,19 +1,40 @@
 <script lang="ts" setup>
+import { SlideoverExampleComponentB } from '#components'
 
 const props = defineProps({
   count: {
     type: Number,
     default: 0
+  },
+  side: {
+    type: String as PropType<'left' | 'right'>,
+    default: 'right'
   }
 })
 
 const emits = defineEmits<{
   close: [];
 }>()
+
+const slideover = useSlideover()
+const anotherCount = ref(0)
+
+
+function openAnotherSlideover () {
+  anotherCount.value += 1
+  
+  const instance = slideover.open(SlideoverExampleComponentB, {
+    count: anotherCount.value,
+    side: 'left',
+    onClose: () => {
+      instance.close()
+    }
```

Comment:
I don't think it's a good idea to oblige the user to write this same code all the time, I'll give you a snippet below